components/brave_player/resources/playerEmbed.tsx

```diff
@@ -0,0 +1,59 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { render } from 'react-dom'
+import styled from 'styled-components'
+
+const EmbedContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+`
+
+const StyledFrame = styled.iframe`
+  max-width: 1000px;
```

Comment:
Yeah, it might be better without mx-width for users withs high-res display. WIth the 1000px width limit, it could be sqaure on the screen with no benefit.